src/Domain/Structure/Neighbors.cpp

```diff
@@ -61,6 +93,11 @@ template <size_t VolumeDim, typename IdType>
 void Neighbors<VolumeDim, IdType>::set_ids_to(
     const std::unordered_set<IdType> new_ids) {
   ids_ = std::move(new_ids);
+  ASSERT(ids_and_orientations_are_consistent(ids_, orientations_),
+         "Some of the new Ids "
+             << ids_
```

Comment:
Bruh, this is _all_ ids, not just new ones. Someone's gonna hate you for this at some point, and it may be you :P

Reply:
they are equivalent in this function, but I'll reword...